### PR TITLE
Fetch photos home

### PR DESCRIPTION
**Pull request by bebzz06**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,18 +18,11 @@ export default function App() {
...
-          <Route path="/user">
-            <User />
-          </Route>
-          <Route path="/favorites">
-            <Favorites />
-          </Route>
+          <Route path="/" exact component={Home} />
+          <Route path="/search/photos/searchterm" component={Search} />
+          <Route path="/explore" component={Explore} />
+          <Route path="/user/username" component={User} />
```
Comment (yoloOnTheBattlefield) [inline src/App.js:24]: path="/search/photos/:searchterm"
path="/user/:username"

File: src/components/PhotoCollection/index.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    componentDidMount() {
+        this.getCollections();
+    }
+    render() {
+        const { collections } = this.state;
+        // if (collections) {
+        //     console.log(collections);
+
+        // }
```
Comment (yoloOnTheBattlefield) [inline src/components/PhotoCollection/index.js:39]: no commented out code

File: src/components/RandomPhotos/index.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+const Photo = styled.img`
+width: 823px;
+height: 531px;
+`
+const Footer = styled.div`
+display: flex;
+justify-content: space-between;
+padding: 27px;
+`
```
Comment (yoloOnTheBattlefield) [inline src/components/RandomPhotos/index.js:44]: make separate file

File: src/pages/Explore/index.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+`;
+const Column = styled.div`
+  width: 33.33%;
+  padding: 24px 15px;
+`;
+const Image = styled.img`
+  width: 100%;
+  border-radius: 5%;
+  padding-bottom: 15px;
+`;
```
Comment (yoloOnTheBattlefield) [inline src/pages/Explore/index.js:23]: make separate file for styles

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (bebzz06) [reply src/App.js:24]: Done

**Pull request merged**